.github/workflows/docs.yml

```diff
@@ -22,7 +22,9 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v4
       with:
-        python-version: 3.8
+        python-version: 3.9
+    - name: Install DisCoPy 1.0.1 from GitHub
+      run: pip install https://api.github.com/repos/discopy/discopy/zipball/pull/148/head
```

Comment:
Need to change this.

Reply:
Same here, we'll release DisCoPy v1.1 asap and remove this.